process-engine/iam
# 🐛 Betafix: Cache by token instead of UserId

## Changes

1. Reduce log spamming, when using test environments
2. Store claim check results by a users token, instead of a userId
3. Increase test coverage

## Issues

Part of https://github.com/process-engine/process_engine_runtime/issues/419

PR: #15

## How to test the changes

- Make claim check requests for the same user, using different tokens with varying permissions
- See that the results are all correct
